registration/include/pcl/registration/transformation_estimation_lm.h

```diff
@@ -216,7 +217,10 @@ namespace pcl
 
         /** \brief The parameterized function used to warp the source to the target. */
         typename pcl::registration::WarpPointRigid<PointSource, PointTarget, MatScalar>::Ptr warp_point_;
-        
+
+        /** \brief L2 regularization coefficients. */
+        VectorX reg_coeff_;
```

Comment:
Note ABI change here

Reply:
I don't understand if you're requesting that I make some kind of note, or if _you're_ noting that this is an ABI change. If there's a change you'd like me to make, please let me know.